apps/condo/migrations/20211122133809-0082_auto_20211122_0838.js

```diff
@@ -0,0 +1,50 @@
+// auto generated by kmigrator
+// KMIGRATOR:0082_auto_20211122_0838:IyBHZW5lcmF0ZWQgYnkgRGphbmdvIDMuMi45IG9uIDIwMjEtMTEtMjIgMDg6MzgKCmZyb20gZGphbmdvLmRiIGltcG9ydCBtaWdyYXRpb25zLCBtb2RlbHMKCgpjbGFzcyBNaWdyYXRpb24obWlncmF0aW9ucy5NaWdyYXRpb24pOgoKICAgIGRlcGVuZGVuY2llcyA9IFsKICAgICAgICAoJ19kamFuZ29fc2NoZW1hJywgJzAwODFfYXV0b18yMDIxMTExOF8xMTU1JyksCiAgICBdCgogICAgb3BlcmF0aW9ucyA9IFsKICAgICAgICBtaWdyYXRpb25zLkFkZEZpZWxkKAogICAgICAgICAgICBtb2RlbF9uYW1lPSdwcm9wZXJ0eScsCiAgICAgICAgICAgIG5hbWU9J2FyZWEnLAogICAgICAgICAgICBmaWVsZD1tb2RlbHMuRGVjaW1hbEZpZWxkKGJsYW5rPVRydWUsIGRlY2ltYWxfcGxhY2VzPTIsIG1heF9kaWdpdHM9MTgsIG51bGw9VHJ1ZSksCiAgICAgICAgKSwKICAgICAgICBtaWdyYXRpb25zLkFkZEZpZWxkKAogICAgICAgICAgICBtb2RlbF9uYW1lPSdwcm9wZXJ0eScsCiAgICAgICAgICAgIG5hbWU9J3llYXJPZkNvbnN0cnVjdGlvbicsCiAgICAgICAgICAgIGZpZWxkPW1vZGVscy5EYXRlRmllbGQoYmxhbms9VHJ1ZSwgbnVsbD1UcnVlKSwKICAgICAgICApLAogICAgICAgIG1pZ3JhdGlvbnMuQWRkRmllbGQoCiAgICAgICAgICAgIG1vZGVsX25hbWU9J3Byb3BlcnR5aGlzdG9yeXJlY29yZCcsCiAgICAgICAgICAgIG5hbWU9J2FyZWEnLAogICAgICAgICAgICBmaWVsZD1tb2RlbHMuRGVjaW1hbEZpZWxkKGJsYW5rPVRydWUsIGRlY2ltYWxfcGxhY2VzPTQsIG1heF9kaWdpdHM9MTgsIG51bGw9VHJ1ZSksCiAgICAgICAgKSwKICAgICAgICBtaWdyYXRpb25zLkFkZEZpZWxkKAogICAgICAgICAgICBtb2RlbF9uYW1lPSdwcm9wZXJ0eWhpc3RvcnlyZWNvcmQnLAogICAgICAgICAgICBuYW1lPSd5ZWFyT2ZDb25zdHJ1Y3Rpb24nLAogICAgICAgICAgICBmaWVsZD1tb2RlbHMuRGF0ZUZpZWxkKGJsYW5rPVRydWUsIG51bGw9VHJ1ZSksCiAgICAgICAgKSwKICAgIF0K
+
+exports.up = async (knex) => {
+    await knex.raw(`
+    BEGIN;
+--
+-- Add field area to property
+--
+ALTER TABLE "Property" ADD COLUMN "area" numeric(18, 2) NULL;
+--
+-- Add field yearOfConstruction to property
+--
+ALTER TABLE "Property" ADD COLUMN "yearOfConstruction" date NULL;
+--
+-- Add field area to propertyhistoryrecord
+--
+ALTER TABLE "PropertyHistoryRecord" ADD COLUMN "area" numeric(18, 4) NULL;
```

Comment:
Arguments of `numeric` in `PropertyHistoryRecord` differs from declared in `Property`
`(18, 2)`
`(18, 4)`

Is it a result of autogeneration?

Reply:
@AntonAL yes, that's autogenerated value